### PR TITLE
Move Ruby Companies link to main area

### PR DESCRIPTION
**Pull request by layerssss**
Everyone look here!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,6 +62,14 @@ <h2>Local Meetups</h2>
   <p>Do you run a meetup that's not listed here? <a href="https://github.com/rubynz/www">Make a pull request</a> to add yours to the site.</p>
 </section>
 <hr />
+
+<section id="ruby-companies">
+  <h2>Ruby/Rails Companies in New Zealand</h2>
+
+  <p>Looking for help with a Ruby website? Here's a <a href="https://github.com/rubynz/www/wiki/Ruby-and-or-Rails-Companies-in-NZ">register of Ruby companies in New Zealand.</a></p>
```
Comment (parndt) [inline index.html]:
```suggestion
  <p>Looking for help with a Ruby project or want to work with Ruby? Here's a <a href="https://github.com/rubynz/www/wiki/Companies-known-to-use-Ruby-in-NZ">register of companies using Ruby in New Zealand.</a></p>
```

File: index.html [hunk 2]
```diff
@@ -62,6 +62,14 @@ <h2>Local Meetups</h2>
   <p>Do you run a meetup that's not listed here? <a href="https://github.com/rubynz/www">Make a pull request</a> to add yours to the site.</p>
 </section>
 <hr />
+
+<section id="ruby-companies">
+  <h2>Ruby/Rails Companies in New Zealand</h2>
```
Comment (parndt) [inline index.html]:
```suggestion
  <h2>Companies known to use Ruby in New Zealand</h2>
```

File: _layouts/default.html [hunk 3]
```diff
@@ -55,7 +55,7 @@ <h3>Ruby in Aotearoa New Zealand</h3>
             <a href="/#current-committee">Current Committee</a>
             <a href="/#past-committee">Past Committee</a>
             <a href="/#slack-moderators">Slack Moderators</a>
-            <a href="https://github.com/rubynz/www/wiki/Ruby-and-or-Rails-Companies-in-NZ">Local Ruby/Rails Companies</a>
+            <a href="/#ruby-companies">Ruby/Rails Companies</a>
```
Comment (parndt) [inline _layouts/default.html]:
```suggestion
            <a href="/#ruby-companies">Companies known to use Ruby in New Zealand</a>
```

**Review by sarahhay7 — CHANGES_REQUESTED** [review]
Love the idea of moving this to the main area ✨ 

I concur with @parndt's suggested changes

**Pull request merged**